<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyQtClient</string>
  </property>
  <widget class="QWidget" name="centralwidget1">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>241</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>451</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar1"/>
 </widget>
 <resources/>
 <connections/>
</ui>
